client/package.json

```diff
@@ -378,7 +378,7 @@
               "mac": "$gobraTools$/server/server.jar"
             }
           },
-          "description": "Paths to the dependencies."
+          "description": "Paths to the dependencies. Only considered for build version 'Local'."
```

Comment:
I would write "They are only considered when the build version is set to `Local`."